components/typography/TextLink.js

```diff
@@ -0,0 +1,20 @@
+
+export default function TextLink({
+    href,
+    className,
+    target = '_self',
+    children
+}) {
+
+    const classNames = `text-secondary-500 underline hover:text-gray-800 font-medium transition ease-in-out duration-300 ${className || ''}`
+
+    return(
+        <>
+        {' '}
```

Comment:
Why we need that empty string? 😄

Reply:
Ok thanks for explanation but I don't think so it's good approach. What if someone want use that link when isn't any pre content? We will add empty, unnecessary space.

🤔  Or just we should add that empty space but only when given prop is enables like `emptyPreSpace={true}`.

Reply:
The idea for the component was to provide a universal way of applying a link within paragraph text. I don't really see this style being used outside of a block element, this is why I named it `TextLink` because it is meant to be applied within the text.

Reply:
I may be stubborn but I don't like it 😅  I've never seen anything like it and I don't think it's necessary. We can leave it but it will have to be improved :)